src/Sulu/Bundle/PageBundle/Domain/Event/PageRemovedEvent.php

```diff
@@ -40,25 +40,37 @@ class PageRemovedEvent extends DomainEvent
      */
     private $pageTitleLocale;
 
+    /**
+     * @var array
+     */
+    private $context;
+
     public function __construct(
         string $pageId,
         string $webspaceKey,
         ?string $pageTitle,
-        ?string $pageTitleLocale
+        ?string $pageTitleLocale,
+        array $context = []
```

Comment:
If looking at the `DomainEvent` class I think this kind of data should be in the `payload` and `getEventPayload` instead of `getEventContext`.

/cc @wachterjohannes what do you think?

Reply:
Let see what the other @sulu/core-team member think about it where to put this kind of data.

Reply:
IMO `payload` should contain the data that is used for updating the entity. So I would say `context` is the way to go here 🙂

Reply:
@alexander-schranz i also agree with the opinion of niklas - we should go with context for this kind of data

Reply:
Should I keep the `$context` constructor argument or replace it with `$pageUrl`?

Reply:
I would keep `$context`, this make it more flexible for the future if we maybe have additional data like a externalId or so on. /cc @sulu/core-team

Reply:
I prefer `$pageUrl`, because it's explicit, typehinted and easier to refactor.
The last time that constructor was modified was 2 years ago.

Reply:
I think there are valid points for both sides. I think I would try to keep it consistent with the other events that have a similar need, but I dont have a too strong opinion 🙂

Reply:
Only the `TagRemovedEvent` class uses an array context contructor argument.

Reply:
We discussed it internal and we will keep the `$context` constructor parameter this way. So you can continue with the other comments. Again thank you for working on this 👍